src/client-scopes/details/ScopeForm.tsx

```diff
@@ -49,9 +49,15 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
   const [openType, setOpenType] = useState(false);
   const { id } = useParams<{ id: string }>();
 
+  const displayOnConsentScreenEnabled =
+    clientScope["attributes"]?.["display.on.consent.screen"];
```

Comment:
You don't have to use the bracket notation here for the `attributes` property.

```suggestion
    clientScope.attributes?.["display.on.consent.screen"];
```